### PR TITLE
Update tdm transform

### PR DESCRIPTION
**Pull request by gwaybio**

### CODE REVIEW

**Comment by gwaybio** [comment]
`scripts/util/transform.r` and `scripts/util/transform.R` are essentially the same script - only minor updates to style and naming conventions.

File: scripts/util/transform.R [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Gregory Way 2016
+# NF1 Inactivation Classifier for Glioblastoma
+# scripts/util/transform.R
+#
+# Transforms the RNAseq distribution to appear more like microarray
+#
+# Usage:
+# Run by transform_rnaseq.py within run_pipeline.sh to transform distribution.
+# The script is run with the following arguments.
```
Comment (amyecampbell) [inline scripts/util/transform.R]: Would be clearer if you replaced "." with ":", but it's not too important

Comment (gwaybio) [reply scripts/util/transform.R]: i agree! updated.

File: scripts/transform_rnaseq.py [hunk 2]
```diff
@@ -23,36 +23,30 @@
...
-####################################
-command = 'R --no-save --args ' + MICRO_FH + ' ' + RNASEQ_FH + \
-          ' ' + OUT_FH + ' < ' + 'scripts/util/transform.r'
+command = 'R --no-save --args {} {} {} < \
+          scripts/util/transform.R'.format(micro_file, rnaseq_file, out_file)
 call(command, shell=True)
 
-X = pd.read_csv(OUT_FH, delimiter='\t', index_col=0)
-zero_one_normalize(X, OUT_FH, method=NORMALIZATION)
+X = pd.read_csv(out_file, delimiter='\t', index_col=0)
```
Comment (amyecampbell) [inline scripts/transform_rnaseq.py]: 'X' and 'OUT_FH' could both be more descriptive, but it's up to you

**Review by amyecampbell — APPROVED** [review]

File: scripts/transform_rnaseq.py (hunk 2 above)
Comment (gwaybio) [reply scripts/transform_rnaseq.py]: I definitely agree - thanks for catching! I've updated the variable names in the most recent push

**Pull request merged**